File: docs/data-tools/codesnippet/CSharp/how-to-update-records-in-a-database_1.cs
            // Locate the row you want to update.
            NorthwindDataSet.RegionRow regionRow;
            regionRow = northwindDataSet.Region.FindByRegionID(1);

            // Assign the new value to the desired column.
            regionRow.RegionDescription = "East";

            // Save the updated row to the database.
            this.regionTableAdapter.Update(this.northwindDataSet.Region);

File: docs/data-tools/codesnippet/VisualBasic/how-to-update-records-in-a-database_1.vb
        ' Locate the row you want to update.
        Dim regionRow As NorthwindDataSet.RegionRow
        regionRow = NorthwindDataSet._Region.FindByRegionID(1)

        ' Assign the new value to the desired column.
        regionRow.RegionDescription = "East"

        ' Save the updated row to the database
        Me.RegionTableAdapter.Update(Me.NorthwindDataSet._Region)